temporian/implementation/numpy/operators/arithmetic/division.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from temporian.implementation.numpy.operators.arithmetic.base import (
+    BaseArithmeticNumpyImplementation,
+)
+from temporian.core.operators.arithmetic import DivisionOperator
+from temporian.implementation.numpy import implementation_lib
+
+
+class DivisionNumpyImplementation(BaseArithmeticNumpyImplementation):
+    """Actual numpy implementation to divide first event by the second one"""
+
+    def __init__(self, operator: DivisionOperator) -> None:
+        super().__init__(operator)
+
+    def _do_operation(self, event_1_data, event_2_data):
+        return event_1_data / event_2_data
```

Comment:
to discuss: do we want alert the user if the division produced `np.nan` values?

cc: @achoum @ianspektor

Reply:
probably, I think torch for example does so, might be the same with other libs

Reply:
Numpy already shows a RuntimeWarning on divide by zero (only once per run), and the result is `np.inf` in that case.